packages/manager/modules/cloud-connect/src/details/statistics/template.html

```diff
@@ -0,0 +1,102 @@
+<section>
+    <form class="form-horizontal">
+        <div class="form-group">
+            <label
+                for="graphType"
+                class="col-xs-3 col-md-2 control-label text-left"
+                data-translate="cloud_connect_stats_type"
+            ></label>
+            <div class="col-xs-9 col-md-4 clearfix">
+                <select
+                    id="graphType"
+                    name="graphType"
+                    class="form-control"
+                    data-ng-disabled="$ctrl.isGraphLoading"
+                    data-ng-change="$ctrl.loadGraphs()"
+                    data-ng-model="$ctrl.graphType"
+                    data-ng-options="'cloud_connect_stats_type_' + type | translate for type in $ctrl.typeOptions"
+                ></select>
+            </div>
+        </div>
+        <div class="form-group">
+            <label
+                for="graphPeriod"
+                class="col-xs-3 col-md-2 control-label text-left"
+                data-translate="cloud_connect_stats_period"
+            ></label>
+            <div class="col-xs-9 col-md-4 clearfix">
+                <select
+                    id="graphPeriod"
+                    name="graphPeriod"
+                    class="form-control"
+                    data-ng-disabled="$ctrl.isGraphLoading"
+                    data-ng-change="$ctrl.loadGraphs()"
+                    data-ng-model="$ctrl.graphPeriod"
+                    data-ng-options="'cloud_connect_stats_period_' + period | translate for period in $ctrl.periodOptions"
+                ></select>
+            </div>
+        </div>
+    </form>
+    <div class="cui-panel">
```

Comment:
Can't we avoid `cui` usage and use bootstrap/ui-kit

Reply:
Why not. Do you know the equivalent for cui-panel, cui-panel__header and cui-panel__body to resolve this ? For your information, I take base on other components for OVHcloud Connect components to have the same look and feel ;-)

Reply:
https://github.com/ovh/manager/blob/master/packages/manager/modules/cloud-styles/src/ui-components/panel/panel.less I guess we could replace with bootstrap/ui-kit header